_data/comments/test-markdown/comment-1545281843190.yml

```
@@ -0,0 +1,7 @@
+_id: bc9dd1e0-0413-11e9-beb4-d1f35c904fc1
+message: "Test again\r\n\r\n> Markdown work?"
```

Test suggestion in beta

```suggestion
message: "Test edit\r\n\r\n> Markdown will work."
```